src/Extensions.Hosting.AsyncInitialization/Extensions.Hosting.AsyncInitialization.csproj

```diff
@@ -12,14 +12,20 @@
     <PackageTags>.net core;hosting;async;initialization</PackageTags>
     <RepositoryUrl>https://github.com/thomaslevesque/Extensions.Hosting.AsyncInitialization</RepositoryUrl>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    <Configurations>Debug;Release;ReleaseLegacyHosting</Configurations>
+		<MSEHostingVersion Condition="'$(Configuration)' == 'ReleaseLegacyHosting'">2.1.0</MSEHostingVersion>
+		<MSEHostingVersion Condition="'$(Configuration)' != 'ReleaseLegacyHosting'">6.0.0</MSEHostingVersion>
```

Comment:
I'm not so sure about this. I understand it's to be able to test with a host that doesn't implement `IAsyncDisposable`, but this configuration is never going to be published, and it won't even be tested in CI. And it adds a lot of noise to the project files.

A simpler way to test this would be to make a wrapper for the host that implements `IDisposable` but not `IAsyncDisposable` (kinda the opposite of the wrapper you already wrote).

Reply:
I'm not really happy with the approach either. You had something similar in the old version, but that used TargetFramework. One could in theory add an additional call to dotnet test and pass the configuration explicitely, but I agree, it's not nice. 
I'll change it and use a wrapper.